benchmark/simple/burrow.json

```diff
@@ -0,0 +1,15 @@
+{
+  "burrow": {
```

Comment:
This is a network configuration file, so it can be move into the corresponding folder 'network/burrow/simplenetwork'. And one section named "caliper" with two subsections 'blockchain' and 'command' could be added into it.  You can modify it according to the sample file '/network/fabric/2org2peer/fabric-simple.json'.